modules/ztp-creating-ztp-crs-for-multiple-managed-clusters.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc
+
+:_content-type: CONCEPT
+[id="ztp-creating-ztp-crs-for-multiple-managed-clusters_{context}"]
+= Installing managed clusters with SiteConfig resources and RHACM
+
+GitOps ZTP uses `SiteConfig` custom resources (CRs) in a Git repository to manage the processes that install {product-title} clusters. The `SiteConfig` CR contains cluster-specific parameters required for installation. It has options for applying select configuration CRs during installation including user defined extra manifests.
+
+The ZTP GitOps plug-in processes `SiteConfig` CRs to generate a collection of CRs on the hub cluster. This triggers the assisted service in {rh-rhacm-first} to install {product-title} on the bare-metal host. You can find installation status and error messages in these CRs on the hub cluster.
+
+You can provision single clusters manually or in batches with ZTP:
+
+Provisioning a single cluster:: Create a single `SiteConfig` CR and related installation and configuration CRs for the cluster, and apply them in the hub cluster to begin cluster provisioning. This is a good way to test your CRs before deploying on a larger scale.
+
+Provisioning many clusters:: Install managed clusters in batches of up to 100 by defining `SiteConfig` and related CRs in a Git repository. ArgoCD uses the `SiteConfig` CRs to deploy the sites. The {rh-rhacm} policy generator creates the manifests and applies them to the hub cluster. This starts the cluster provisioning process.
```

Comment:
The batch size here is out of date. The current recommendation is batches of up to 400.